src/main/scala/com/actian/spark_vector/colbuffer/decimal/DecimalIntColumnBuffer.scala

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
I am merging this branch to master. The comments that are still left are going to be addressed as a new pull request (working on it).